packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.html

```diff
@@ -28,11 +28,14 @@
 
   <amplify-slot name="sign-in-footer">
     <button
-      amplify-button
-      fontWeight="normal"
-      size="small"
-      variation="link"
       (click)="authenticator.toResetPassword()"
+      class="amplify-button amplify-field-group__control"
+      data-fullwidth="true"
+      data-size="small"
+      data-variation="link"
+      data-amplify-button=""
```

Comment:
Quick question, why did you need to use `data-*`? `<button amplify-button fullWidth="true" />` to my knowledge.

https://github.com/aws-amplify/amplify-ui/blob/6dd6bf915b83e623dc9e526be7805dee15b9af8f/packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.ts#L8-L12

Reply:
Honestly, this was a copy paste from the Vue implementation.

I completely missed that this was an `amplify-button` directive!

I'll attempt this again, BRB...

Reply:
That works!  Sorry @wlee221 !

Fixed via cb2577ea31f88d42bface6a8b2f55c38110b8856